src/main.rs

```diff
@@ -100,6 +100,10 @@ struct Args {
     /// Execution mode to start rollup boost with
     #[arg(long, env, default_value = "enabled")]
     execution_mode: ExecutionMode,
+
+    /// percentage multipler to the local payload gas used for choosing payloads
+    #[arg(long, env, default_value = "0")]
+    l2_boost_factor: u64,
```

Comment:
I'm not sure l2_boost_factor is a very descriptive name. Maybe we can go with something like `--builder-gas-requirement=0.95` and parse a float here instead?